package.json

```diff
@@ -111,15 +111,15 @@
     "@tsconfig/svelte": "^3.0.0",
     "@types/cache-manager": "^4.0.1",
     "@types/express": "^4.17.13",
-    "@types/jest": "28.1.7",
+    "@types/jest": "28.1.8",
     "@types/luxon": "^3.0.0",
-    "@types/node": "18.7.12",
+    "@types/node": "18.7.13",
     "@types/react": "18.0.17",
     "@types/react-dom": "18.0.6",
     "@types/react-router-dom": "5.3.3",
-    "@typescript-eslint/eslint-plugin": "5.34.0",
+    "@typescript-eslint/eslint-plugin": "5.35.1",
     "@typescript-eslint/parser": "5.34.0",
-    "astro": "^1.0.0-rc.8",
+    "astro": "^1.1.3",
```

Comment:
@mschilling Did this not require any migrations to work correctly?
Reason I left this dependency upgrade open, to take the time to go through the migration guide (when I have time 🙂)

Reply:
I didn't look into the details (other than also working on another migratie: 0.23 to ~1.0). Did verify `nx build` and `nx serve` for the project and it seems to work as expects.